src/components/actions/coreActions.ts

```diff
@@ -0,0 +1,493 @@
+      sequence: -3,
+      condition: {
+        operator: 'AND',
+        children: [
+          {
+            operator: 'objectProperties',
+            children: ['applicationData.reviewData.isLastLevel'],
+          },
+          {
+            operator: '=',
+            children: [
+              {
+                operator: 'objectProperties',
+                children: ['applicationData.reviewData.latestDecision.decision'],
+              },
+              'CONFORM',
+            ],
+          },
+        ],
+      },
+      parameter_queries: {},
+    },
+    // Change status to "CHANGES_REQUIRED" if last level and decision = LOQ
+    {
+      code: 'changeStatus',
+      path: '../plugins/action_change_status/src/index.ts',
+      name: 'Change Status',
+      trigger: 'ON_REVIEW_SUBMIT',
+      event_code: null,
+      sequence: -2,
+      condition: {
+        operator: 'AND',
+        children: [
+          {
+            operator: '=',
+            children: [
+              {
+                operator: 'objectProperties',
+                children: ['applicationData.reviewData.latestDecision.decision'],
+              },
+              'LIST_OF_QUESTIONS',
+            ],
+          },
+          {
+            operator: 'objectProperties',
+            children: ['applicationData.reviewData.isLastLevel'],
+          },
+        ],
+      },
+      parameter_queries: {
+        isReview: false,
+        newStatus: 'CHANGES_REQUIRED',
+      },
+    },
+    // Change outcome accordingly if final level and stage is CONFORM or
+    // NON_CONFORM
+    {
+      code: 'changeOutcome',
+      path: '../plugins/action_change_outcome/src/index.ts',
+      name: 'Change Outcome',
+      trigger: 'ON_REVIEW_SUBMIT',
+      event_code: null,
+      sequence: -1,
+      condition: {
+        operator: 'OR',
+        children: [
+          {
+            operator: 'AND',
+            children: [
+              {
+                operator: '=',
+                children: [
+                  {
+                    operator: 'objectProperties',
+                    children: ['applicationData.reviewData.latestDecision.decision'],
+                  },
+                  'CONFORM',
+                ],
+              },
+              {
+                operator: 'objectProperties',
+                children: ['applicationData.reviewData.isLastStage'],
+              },
+            ],
+          },
+          {
+            operator: 'AND',
+            children: [
+              {
+                operator: '=',
+                children: [
+                  {
+                    operator: 'objectProperties',
+                    children: ['applicationData.reviewData.latestDecision.decision'],
+                  },
+                  'NON_CONFORM',
+                ],
+              },
+              {
+                operator: 'objectProperties',
+                children: ['applicationData.reviewData.isLastLevel'],
+              },
```

Comment:
This condition is a bit different than what is intended to be

Reply:
Okay, I've updated it to be a bit simpler (although I think the logic is actually the same, just in a more concise expression): e76a5c86

Did you have something else in mind?

Reply:
No... It isn't there yet. I know that Core-actions had a different version (not fully operational). I'll add my suggestion and start testing this on `core_templates` snapshot